lisa/tools/service.py

```diff
@@ -120,3 +127,11 @@ def _check_service_running(self, name: str) -> bool:
         return (
             "could not be found" not in cmd_result.stdout and 0 == cmd_result.exit_code
         )
+
+
+def _check_error_codes(cmd_result: ExecutableResult, error_code: int = 0) -> None:
+    if error_code:
```

Comment:
I guess it's ok to include two `0`. So direct use `[0, error_code]` is simple and ok.